packages/manager-react-components/src/components/Modal/Modal.component.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {
+  OdsButton,
+  OdsModal,
+  OdsText,
+  OdsSpinner,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_MODAL_COLOR,
+  ODS_BUTTON_COLOR,
+  ODS_TEXT_PRESET,
+  ODS_SPINNER_SIZE,
+} from '@ovhcloud/ods-components';
+
+export enum ModalTypeEnum {
+  'info' = 'info',
+  'help' = 'help',
+  'success' = 'success',
+  'warning' = 'warning',
+  'error' = 'error',
+}
+
+export interface ModalProps {
+  /** Title of modal */
+  heading?: string;
+  /** Type of modal */
+  type?: ModalTypeEnum;
+  /** Is loading state for display a spinner */
+  isLoading?: boolean;
+  /** Label of primary button */
+  primaryLabel?: string;
+  /** Is loading state for primary button */
+  isPrimaryButtonLoading?: boolean;
+  /** Is disabled state for primary button */
+  isPrimaryButtonDisabled?: boolean;
+  /** Action of primary button */
+  onPrimaryButtonClick?: () => void;
+  /** Label of secondary button */
+  secondaryLabel?: string;
+  /** Is loading state for secondary button */
+  isSecondaryButtonDisabled?: boolean;
+  /** Is loading state for secondary button */
+  isSecondaryButtonLoading?: boolean;
+  /** Is disabled state for secondary button */
+  onSecondaryButtonClick?: () => void;
+  /** Is dismissible action */
+  onDismiss?: () => void;
+  /** Is modal open state */
+  isOpen?: boolean;
+  /** Children of modal */
+  children?: React.ReactElement;
```

Comment:
Prefer use `React.PropsWithChildren<YourProps>` type instead of declaring it like this, or put a `React.ReactNode` type

Reply:
I put React.ReactNode, so it's fixed.